awx/provisioning/playbook.yml

```diff
@@ -11,7 +11,8 @@
     awx_version: "devel"
     nodejs_version: "6.x"
     pip_install_packages:
-      - name: docker-py
+      - name: docker
+      - name: docker-compose
```

Comment:
Docker compose will automatically be installed by the `geerlingguy.docker` role (by default), so in this case just changing the package to `docker` should be sufficient. If you can make that change, I'm happy to merge this in. Thanks!